class-lifterlms.php

```diff
@@ -420,4 +421,25 @@ public function localize() {
 
 	}
 
+	/**
+	 * Filter Hook to enable the Favorite feature.
+	 *
+	 * @since [version]
+	 *
+	 * @return boolean
+	 */
+	public function is_favorites_enabled() {
```

Comment:
we don't have anything like that there, I would avoid having a method here.
let's go with a simple function in `functions/llms.functions.favorite.php`

Reply:
moved.